front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -353,8 +353,16 @@
               justify-content: right;
 
               .after-midnight {
-                max-width: 1rem;
-                margin: 0 0.5rem;
+                width: 1.5rem;
+                height: 1.5rem;
+                color: var(--info60);
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                margin: 0 0.25rem;
+                span {
+                  padding-bottom: 0.25rem;
+                }
```

Comment:
Could we unify this with `not-honored-or-too-fast`? Seems like it could share most of this?

Reply:
You're right, I've pushed a solution